src/platform_impl/windows/event_loop.rs

```diff
@@ -1546,11 +1557,11 @@ unsafe fn public_window_callback_inner<T: 'static>(
                     let mut device_rect = mem::MaybeUninit::uninit();
                     let mut display_rect = mem::MaybeUninit::uninit();
 
-                    if (GetPointerDeviceRects(
+                    if !GetPointerDeviceRects(
                         pointer_info.sourceDevice,
                         device_rect.as_mut_ptr(),
                         display_rect.as_mut_ptr(),
-                    )) == 0
+                    ) == true.into()
```

Comment:
Semantic change (negative comparison to `true` instead of positive comparison to `false`)

At this point I'm gonna stop with these BOOL changes, but they're present a lot of places below